### ISSUE TITLE
Update to 1.13.2

### ISSUE THREAD

**Issue by Spuito65** [TheBusyBiscuit/ExoticGarden#48]
Will it be updated ? Or at least made the alpha/beta playable without compiling like Slimefun ?

**Comment by AeSix** [comment]
There is currently a CS-CoreLib, SlimeFun4 and ExoticGarden which load and mostly work on 1.13.2.
There are still issues however. I wouldn't update a production server yet if you can hold off on that.

**Comment by SoSeDiK** [comment]
Done :)
Report if there are any issues.

**Comment by AeSix** [comment]
Updated all the plugins and testing now.  I'll report back any issues :)
(Slimefun, CS-CoreLib, ExoticGarden, ChestTerminals, and Luckyblocks)

**Comment by AeSix** [comment]
There is an issue you might can look at, so far.  I created the ticket under SlimeFun, as thats what the error is pointing to.  You can find it here: https://github.com/TheBusyBiscuit/Slimefun4/issues/862

**Comment by mrcoffee1026** [comment]
Having trouble updating from 1.12.2 to 1.13.2.  On my local build... running the same Spigot or Paper version (v1_13_R2) as well as CS-CoreLib (vDev - 36), Slimefun (vDEV 91) and ExoticGarden (vDEV - 8)... they both do a huge amount of logging on startup that look like this:

```
[11:09:04] [Server thread/INFO]: [Slimefun] New block data (world;-8444;64;3679): {"id":"BLUEBERRY_BUSH"}
[11:09:04] [Server thread/INFO]: [Slimefun] Ignoring duplicate block @ 10301, 64, -18648
[11:09:04] [Server thread/INFO]: [Slimefun] Old block data: {"id":"BLUEBERRY"}
[11:09:04] [Server thread/INFO]: [Slimefun] New block data (world;10301;64;-18648): {"id":"BLUEBERRY_BUSH"}
[11:09:04] [Server thread/INFO]: [Slimefun] Ignoring duplicate block @ -17928, 64, 21210
[11:09:04] [Server thread/INFO]: [Slimefun] Old block data: {"id":"BLUEBERRY"}

```
Thing is this seems to be a procedure that actually finishes... so not sure why it needs to this EVERY startup now, but whatever.  Oddly after my local build shows up, all of my blueberry bushes and things actually work.  On my server... they do not.  Even though the data files are the same (I actually copied from my local drive to the server to be certain... same world files, same everything... same coordinates for all blocks).  Any ideas why the server copy would not be working?  Even when I test the blocks with the debug fish it acts like it has no data even though I verified the blocks address exists for the block I'm checking.   This seems limited to the plants that existed in the world prior to the update only... things lime machines/cargo nodes... all that stuff seems fine.
